sdk/formrecognizer/Azure.AI.FormRecognizer/tests/FormRecognizerClient/FormRecognizerClientLiveTests.cs

```diff
@@ -58,13 +58,9 @@ private FormRecognizerClient CreateInstrumentedFormRecognizerClient(bool useToke
         public async Task FormRecognizerClientCanAuthenticateWithTokenCredential()
         {
             var client = CreateInstrumentedFormRecognizerClient(useTokenCredential: true);
-            RecognizeContentOperation operation;
-
             using var stream = FormRecognizerTestEnvironment.CreateStream(TestFile.ReceiptJpg);
-            using (Recording.DisableRequestBodyRecording())
-            {
-                operation = await client.StartRecognizeContentAsync(stream);
-            }
```

Comment:
We don't want to enable body recording here. The reason for disabling it in tests that use Stream is so we don't end up with huge recording files (to avoid making the repo too big).

We only want to re-enable body recording in training tests that send a SAS URL in the request body. I believe I marked all of these with a:
```
// TODO: sanitize body and enable body recording here.
```

Reply:
will undo